doc/getting_started.md

```diff
@@ -0,0 +1,25 @@
+# Getting started with PacSim
+
+To use PacSim, you need to create a message converter node to match your own interfaces with the simulator. You will need to advertise and publish to some topics. Note that some topics are not directly used in the simulator.
```

Comment:
for the last sentence:

Note that some topics are not used in the simulator with the default vehicle model provided.